profile_xf05id1/startup/80-areadetectors.py

```diff
@@ -20,7 +20,7 @@ class BPMCam(SingleTrigger, AreaDetector):
     image = C(ImagePlugin, 'image1:')
 
     tiff = C(SRXTIFFPlugin, 'TIFF1:',
-             write_path_template='/epicsdata/bpm1-cam1/')
+             write_path_template='/epicsdata/bpm1-cam1/2016/2/17/')
```

Comment:
There is no need to hard-code this date. `write_path_template` accepts` Python date format strings like '%Y/%m/%d'.